run-emulator.sh

```diff
@@ -18,6 +18,27 @@ if [ x"$GDBSERVER" != x"" ]; then
     TAIL_ARGS="$TAIL_ARGS -s -S"
 fi
 
+dns_servers=""
+if [ x"$EMU_DNS_SERVER" != x"" ]; then
```

Comment:
I don't think we use the 'EMU' prefix anywhere else. I suggest to rename this to 'B2G_DNS_SERVER'.

Unfortunately the whole file uses a mixture of capital- an non-capical-case variable names, so there's no clear rule on variable naming.